### DISCUSSION TITLE
Noob-Questions before i start getting deeper into this great project

### DISCUSSION THREAD

**Discussion by guevara777** [Neuroquila-n8fall/JunkersControl#44] [Q&A] [answered]
First things first: Thank you for this great project. I want to try using it with my Junkers ZBS 16 83 S2 MA.

I have to questions before start digging deeper into it. Maybe someone here can help me. Thank you.

1.) I do have the "1-2-4-6-A-F"-Interface on my device, but no TA270 oder TA300 connected to it. Do i need a connected room thermostat or is this optional?

2.) The wiring between the Interface and the niren-board has to be this way?
```
"1-2-4-6-A-F"-Interface <--------> Niren MCP2515 + TJA1050
"2" (CAN HIGH)          <--------> "H" (high)
"6" (CAN LOW)           <--------> "L" (high)
```
or is it "4" (CAN LOW). This picture is confusing to me:
![image](https://github.com/Neuroquila-n8fall/JunkersControl/assets/29648553/5e5e173f-fe2e-46ed-abce-d83a4361f7df)

**Comment by guevara777** [comment]
![image](https://github.com/Neuroquila-n8fall/JunkersControl/assets/29648553/4c8d341d-49a9-46f9-90dd-ac756504ef8b)

It seems i only have a TA 211 e-Module with 1-2-4-A-F, not a BM1 oder BM2-Module. 
Is it possible to add a BM1 oder BM2-Module to this setup?

**Reply by Neuroquila-n8fall** [reply, marked as answer]
Hi there :) 

It appears as if the module you are currently using does not have a CAN interface built in. For this you would need to swap that manual thing out and replace it with a BM1 bus module first. Then you may use the CAN interface as described ;)

**Reply by dmahhhh** [reply]
> ![image](https://github.com/Neuroquila-n8fall/JunkersControl/assets/29648553/4c8d341d-49a9-46f9-90dd-ac756504ef8b)
> 
> 
> 
> It seems i only have a TA 211 e-Module with 1-2-4-A-F, not a BM1 oder BM2-Module. 
> 
> Is it possible to add a BM1 oder BM2-Module to this setup?

Hey, did you manage to swap the TA-211 for the BM1 module? I'm looking forward to doing the same.

**Reply by dmahhhh** [reply]
I manage to get it run. I replaced the TA 211E with BM1 and it worked perfectly!
Thanks for this great project! Thumbs up 

My device list:
- ESP32-D from [AliExpress](https://de.aliexpress.com/item/1005006449303342.html?spm=a2g0o.productlist.main.5.689e1509p2qNk2&algo_pvid=a8184b5d-2bd3-4c83-bb5f-448d62f5045d&algo_exp_id=a8184b5d-2bd3-4c83-bb5f-448d62f5045d-4&pdp_ext_f=%7B%22order%22%3A%221937%22%2C%22eval%22%3A%221%22%2C%22fromPage%22%3A%22search%22%7D&pdp_npi=6%40dis%21EUR%2113.52%210.99%21%21%21108.59%217.97%21%40210390c217618969195257548eb99f%2112000039288143047%21sea%21DE%210%21ABX%211%210%21n_tag%3A-29910%3Bd%3A71435f7c%3Bm03_new_user%3A-29895%3BpisId%3A5000000187470434&curPageLogUid=Qw6c046Db42b&utparam-url=scene%3Asearch%7Cquery_from%3A%7Cx_object_id%3A1005006449303342%7C_p_origin_prod%3A)
- CAN Shield MCP2515 "NIREN" 8Mhz (HW-184) from [Amazon.de](https://www.amazon.de/dp/B0D2PV13CG?ref=ppx_yo2ov_dt_b_fed_asin_title&th=1) 
- 8-32V to 5V StepDown Converter with USB-C from [Amazon](https://www.amazon.de/dp/B0BZ7S1T6X?ref=ppx_yo2ov_dt_b_fed_asin_title)
